apps/organizations/models.py

```diff
@@ -335,6 +339,96 @@ def build_notification_message__declined(self):
         return msg
 
 
+class BaseAbstractOrganizationJoinRequest(HtkBaseModel):
+    organization = fk_organization(related_name='join_requests', required=True)
+    user = fk_user(
+        related_name='organization_join_requests',
+    )
+    email = models.EmailField(
+        blank=True, null=True, default=None
+    )  # the email where the request is sent
+    token = models.UUIDField(default=uuid.uuid4, unique=True)
+    accepted = models.BooleanField(
+        default=None, null=True
+    )  # True: accepted, False: declined, None: not responded yet
+    timestamp = models.DateTimeField(auto_now_add=True)
+    responded_at = models.DateTimeField(blank=True, null=True, default=None)
+    request_message = models.TextField(
```

Comment:
Not sure if this is needed.

Reply:
Okay, will remove. I thought it could be cool to provide an optional introduction or bio when requesting to join.

Reply:
I will just remove it.